src/Tizen.Wearable.CircularUI.Forms.Renderer/CircularUI.cs

```diff
@@ -44,5 +48,27 @@ public static void Init(string apiKey)
 
             Init();
         }
+
+        public static void AddAppPropertyChangedHandler(this Application application)
```

Comment:
Name is ambiguous, I do like to more descriable naming

Reply:
agreed. :-P